scripts/screeps_rl_experiment_card.py

```
@@ -2295,6 +2300,19 @@ def validate_policy_gradient_update(raw: Any) -> None:
     bounded_step = first_present(raw, ("bounded_integer_step", "boundedIntegerStep"))
     if bounded_step is not True:
         raise CardValidationError("policy_gradient.policy_update.bounded_integer_step must be true")
+    trust_gate = first_present(raw, ("gradient_trust_gate", "gradientTrustGate"))
+    if not isinstance(trust_gate, dict):
+        raise CardValidationError("policy_gradient.policy_update.gradient_trust_gate must be a JSON object")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)